explorer/src/api/graphql/scalars.rs

```diff
@@ -87,13 +90,13 @@ impl ScalarType for PoolId {
     }
 }
 
-pub struct Value(pub blockcfg::Value);
+pub struct Value(pub InternalValue);
```

Comment:
I'm not sure if I understand. The `InternalValue` alias is only to be able to import it because the newtype wrapper has the same name. I could fully qualify it though, but `chain_impl_mockchain::value::Value` seemed rather long (especially because it's referenced 3 times in this file). Maybe `LibsValue` would be clearer?

Reply:
oh sorry, it's perfectly fine then